src/core/dbus-manager.c

```diff
@@ -2487,6 +2537,7 @@ const sd_bus_vtable bus_manager_vtable[] = {
         SD_BUS_PROPERTY("Environment", "as", property_get_environment, 0, 0),
         SD_BUS_PROPERTY("ConfirmSpawn", "b", bus_property_get_bool, offsetof(Manager, confirm_spawn), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("ShowStatus", "b", property_get_show_status, 0, 0),
+        SD_BUS_WRITABLE_PROPERTY("WritableShowStatus", "s", property_get_show_status_string, property_set_show_status_string, 0, 0),
```

Comment:
i don't like this I must say. I think it would be nicer to add a method then. SetShowStatus() which optionally takes the empty string